### DISCUSSION TITLE
Model Icons in Light/Dark mode

### DISCUSSION THREAD

**Discussion by TiancongLx** [open-webui/open-webui#7819] [Q&A]
I'm facing a challenge in my admin console's model settings, where I can upload icons for each model.

For multicolored icons such as Claude there is no problem, but for monochrome icons (black or white), they disappear against the background when switching between light and dark modes.

What are some best practices or solutions to handle this issue with monochrome icons?

**Comment by kpatdev** [comment]
Running into this now as well. 

It would be awesome if we could just have a switch called invert that just inverts the icon of that specific model. Would allow for granularity which is needed since some icons like gemini or claude are colorful but openai/ollama are white or black. 

Any thoughts on this?

**Reply by TiancongLx** [reply]
Any thoughts on this? @tjbck

**Reply by recrudesce** [reply]
> Running into this now as well.
> 
> It would be awesome if we could just have a switch called invert that just inverts the icon of that specific model. Would allow for granularity which is needed since some icons like gemini or claude are colorful but openai/ollama are white or black.
> 
> Any thoughts on this?

If you want a manual process to "invert" a logo by toggling a switch in the model settings, why not just replace the logo with one that fits the dark mode feature ?

**Reply by kpatdev** [reply]
Sorry I didn't explain it well but what I meant was that switch inverts that logo when the mode switches.

So I have a black png or svg for OpenAI's logo. If that "invert" switch is toggled for that model, when I switch to dark mode, the logo would be inverted.

But for Gemini, it's a color logo so for those models I don't want inverted colors when I'm switching themes.

Maybe a better way to label the switch would be "Invert for dark mode" or something along those lines.

I'm doing this currently for a site (nextjs) so that's why I mentioned it.
